agent/src/beerocks/slave/gate/1905_beacon_query_to_vs.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#ifndef _1905_BEACONB_QUERY_TO_VS_H_
+#define _1905_BEACONB_QUERY_TO_VS_H_
+
+#include <beerocks/tlvf/beerocks_message_monitor.h>
+#include <tlvf/CmduMessageRx.h>
+#include <tlvf/wfa_map/tlvBeaconMetricsQuery.h>
+
+namespace gate {
```

Comment:
Currently everything is in namespace beerocks (or mapf), so please also put this in namespace beerocks::gate.